src/index.html

```diff
@@ -2,16 +2,182 @@
+          <input type="number" name="years" 
+          min="1" max="100" 
+          value="12" id="Years"
+          >
+
+        </div>
+
+        <div class="form-field">
+          <label for="DateOfBirth">
+            Full date of birth:
+          </label>
+          <input type="date" name="dateofbirth"
+          id="DateOfBirth"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="term-of-the-aggrement">
+        I accept the term of the agreement
+          </label>
+          <input type="checkbox" name="terms" 
+          id="term-of-the-aggrement" required
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="E-mail">
+            E-mail:
+          </label>
+          <input type="email" name="email" 
+          placeholder="email@example.com" required
+          id="E-mail"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="Password">
+            Password:
+          </label>
+          <input type="password" name="password" 
+          id="Password" required
+          >
+        </div>
+      </fieldset>
+    
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+      Do you love cats?
+          <label>
+            <input type="radio" name="cats">
+        Yes
+          </label>
+          <label>
+            <input type="radio" name="cats">
+        No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="favorite-color">
+            What is your favorite color?
+          </label>
+          <input type="color" name="color"
+          id="favorite-color"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="what-time">
+            What time do you go to bed?
+          </label>
+          <input type="time" name="time"
+          step="1" id="what-time"
+          >
+        </div>
+
+        <label class="form-field">
+      What are your favorite brand of cars?
+
+          <select name="color" required 
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option value=" "></option>
+          </select>
+        </label>
+
+        <div class="form-field">
+          <p class="rate">How do you rate our work?
+      
+            <input type="range" name="range" 
+            min="1" max="10" 
+            value="1"
+            >
+          </p>
+        </div>
+      </fieldset>
+    
+      <fieldset>
+        <legend>Additional info:</legend>
+
+        <div class="form-field">
+          <label for="comments">
+            Comments:
+          </label>
+          <textarea> 
+            name="Comments:" 
+            id="comments"
+          </textarea>
```

Comment:
```suggestion
          <textarea 
            name="Comments:" 
            id="comments"
         ></textarea>
```

![image](https://user-images.githubusercontent.com/80700247/226932266-fe599867-7f2f-4ad4-8c47-bc295c42c675.png)

It should fix extra spaces in textarea field